src/components/accessories/admin/users/editGroup/EditGroup.tsx

```diff
@@ -68,46 +82,88 @@ export const EditGroup = () => {
     resetForm,
     errors,
     touched,
-    values,
   } = useFormik({
     initialValues: state,
     validationSchema: userGroupSchema(t),
     onSubmit: (values: UserGroupDTO) => {
       dispatch(updateUserGroup(values));
-      for (let index = 0; index < permissionsStack.length; index++) {
-        dispatch(
-          updatePermission({
-            id: permissionsStack[index].id,
-            permissionDTO: permissionsStack[index],
-          })
+
+      // this is applying to all permissions, but we may want to only apply to the ones that have changed
+      // using the updatedPermissionsStack array
+      // this is open to debate
```

Comment:
To discuss further

Reply:
we are going to open an issue for this and, as discussed, the old endpoint should remain active